js.go

```diff
@@ -166,8 +169,9 @@ const (
 	defaultAccountCheck = 20 * time.Second
 )
 
-// JetStream returns a JetStreamContext for messaging and stream management.
-func (nc *Conn) JetStream(opts ...JSOpt) (JetStreamContext, error) {
+// BindJetStream returns a JetStreamContext for messaging and stream management that will NOT create
+// underlying objects like stream or durable. It assumes they exist already.
```

Comment:
I don't think this is necessary here.

Reply:
Reading the diff, this looks like it's on an internal API.  No problem keeping it if you feel it's important, it's just identical to the exported API below on line 191.

Reply:
sorry my bad. removed